webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -117,4 +118,15 @@ export const getCVFilterRules = (filterId, params) => get({
   url: api.getApiUrl(`/content_view_filters/${filterId}/rules`),
 });
 
+export const getContentViewComponents = (cvId, params) => {
+  const apiParams = { ...params };
+  const apiUrl = `/content_views/${cvId}/content_view_components`;
+  return get({
+    key: cvDetailsComponentKey(cvId),
+    params: apiParams,
+    errorToast: error => __(`Something went wrong while retrieving the content view components! ${error}`),
```

Comment:
Does this have to be a function?

Reply:
The error toast?

Reply:
Yeah, I was just curious why the value of `errorToast` is a function.

Reply:
It's cause we need the error as a param to form the error message.

We do need to parse through error messages better using the helper function we are using elsewhere but not in Detail Actions. Will push an update to do so..

https://github.com/Katello/katello/blob/628ee7ed2a40ec1ea33c1d84bd940035ad34bdec/webpack/utils/helpers.js#L26